public/index.html

```diff
@@ -51,5 +51,8 @@ <h5 class="collaborators">{{colaborators}}</h5>
     <script src="https://code.jquery.com/jquery-3.2.1.min.js"></script>
     <script src="https://cdn.jsdelivr.net/handlebarsjs/4.0.8/handlebars.min.js"></script>
     <script src="app.js"></script>
+    <script>
+      app.page.initPage();
```

Comment:
Depending on the app, you may still wish/need to use `$(document).ready(app.page.initPage)` or simply `$(app.page.initPage)`.